components/dashboards-web-component/src/designer/components/PageCard.jsx

```diff
@@ -151,6 +151,18 @@ class PageCard extends Component {
                                 disabled={page.id === this.props.landingPageId}
                                 onCheck={() => this.props.updateLandingPage(this.props.page.id)}
                             />
+                            {
+                                /* Show hide page option only for non-landing page pages. */
+                                page.id !== this.props.landingPageId &&
+                                <Checkbox
+                                    label={<FormattedMessage id="designer.hide.page" defaultMessage="Hide Page" />}
```

Comment:
Since this checkbox is in a page car, how about just `"Hide"`?

Reply:
Instead of "Hide",  better to have "Hide Page" since it provides some sense to the end-user about its functionality.

Reply:
Ok